lib/appium_lib/ios/helper.rb

```diff
@@ -55,25 +55,25 @@ def fix_space s
         visible = (type.downcase.include?(class_name)).to_s if class_name
         if visible && visible == 'true'
           if name == label && name == value
-            puts "#{type}" if name || label || value || hint
-            puts "   name, label, value: #{name}" if name
-            puts "   hint: #{hint}" if hint
+            Appium::Logger.info "#{type}" if name || label || value || hint
```

Comment:
This function is meant to output to stdout so using the logger here doesn't make sense.